### PR TITLE
Google analytics

### PR DESCRIPTION
**Pull request by dnaranjo89**

### CODE REVIEW

File: server/context_processors.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+def google_analytics(request):
+    """
+    Use the variables returned in this function to
+    render your Google Analytics tracking code template.
+    """
+    ga_prop_id = getattr(settings, 'GOOGLE_ANALYTICS_PROPERTY_ID', False)
+    if ga_prop_id:
+        return {
+            'GOOGLE_ANALYTICS_PROPERTY_ID': ga_prop_id,
+        }
```
Comment (Mariocj89) [inline server/context_processors.py]: missing else

File: web/google_analytics.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    }
+    if label:
+        params_dict['el'] = label
+    if value:
+        params_dict['ev'] = value
+
+
+    params = urlencode(params_dict)
+    connection = HTTPConnection('www.google-analytics.com')
+    connection.request('POST', '/collect', params)
```
Comment (Mariocj89) [inline web/google_analytics.py]: what happens if any of this throws an exception?

**Comment by Mariocj89** [comment]
Would be nice to track user creation as well :) :goat:

File: server/context_processors.py (hunk 1 above)
Comment (dnaranjo89) [inline server/context_processors.py]: It has a return {} in the following line

**Comment by dnaranjo89** [comment]
It was coming in the next PR, but I will add it here

**Pull request closed without merging**